be/src/services/entity.service.ts

```diff
@@ -12,6 +12,10 @@ export class EntityService {
     return persistenceService.db().collection('entities');
   }
 
+  static countCollectionItems(): Promise<number> {
+    return this.collection().countDocuments();
```

Comment:
We need the `filter` + `options` from #136 of this file
https://github.com/TailoredMediaProject/tm-knowledge-store/pull/32/files#diff-86cfaede7e357c33fd6d85f5006ac280063975623322280079f324635bceac19R136
applied here if present but without the paging.